airbyte-integrations/connectors/source-kyriba/source_kyriba/spec.json

```diff
@@ -0,0 +1,50 @@
+{
+  "documentationUrl": "https://docsurl.com",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Kyriba Spec",
+    "type": "object",
+    "required": ["domain", "username", "password"],
+    "additionalProperties": false,
+    "properties": {
+      "domain": {
+        "type": "string",
+        "description": "Kyriba domain",
+        "title": "Domain",
+        "examples": ["demo.kyriba.com"],
+        "pattern":"^[a-zA-Z0-9._-]*\\.[a-zA-Z0-9._-]*\\.[a-z]*"
+      },
+      "version": {
+        "type": "integer",
+        "description": "API version to be used in the URL.",
+        "title": "Version",
+        "examples": [1, 2]
+      },
+      "username": {
+        "type": "string",
+        "description": "Username to be used in basic auth",
+        "title": "Username"
+      },
+      "password": {
+        "type": "string",
+        "description": "Password to be used in basic auth",
+        "title": "Password",
+        "airbyte_secret": true
+      },
+      "start_date": {
+        "type": "string",
+        "description": "The date the sync should start from.",
+        "title": "Start Date",
+        "examples": ["2021-01-10"],
+        "pattern": "^\\d{4}\\-(0[1-9]|1[012])\\-(0[1-9]|[12][0-9]|3[01])$"
+      },
+      "end_date": {
+        "type": "string",
+        "description": "The date the sync should end.",
```

Comment:
If set to empty will sync data to most recent date. (maybe adding this to become clear to users)